Sources/Playbook/Components/Tab Bar/TabBarCatalog.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  TabBarCatalog.swift
+//
+//
+//  Created by Rachel Radford on 12/20/23.
+//
+
+import SwiftUI
+
+public struct TabBarCatalog: View {
+  @State var selectedTab: String = ""
+  @State var selectedTab1: String = ""
+  @State var selectedTab2: String = ""
+  @State var selectedTab3: String = ""
+  @State var selectedTab4: String = ""
+  public var body: some View {
+    ScrollView {
+      VStack(spacing: Spacing.xSmall) {
+        Text("Default")
```

Comment:
This code is repetitive, What about creating a component with this details like our PBDoc?

Reply:
ok, maybe that could be a separate story? Since this kits catalog isnt structured like our others? We'd need to think of the standard structure of it, I think